DataPlotly/gui/plot_settings_widget.py

```diff
@@ -635,6 +635,21 @@ def refreshWidgets(self):  # pylint: disable=too-many-statements,too-many-branch
         self.color_scale_combo.addItem(self.tr('RedOrangeYellow'), 'YIOrRd')
         self.color_scale_combo.addItem(self.tr('DeepblueBlueWhite'), 'YIGnBu')
         self.color_scale_combo.addItem(self.tr('BlueWhitePurple'), 'Picnic')
+        
+        self.color_scale_data_defined_in.clear()
+        self.color_scale_data_defined_in.addItem(self.tr('Grey Scale'), 'Greys')
```

Comment:
This could be improved by iterating through a list or ordered dict and adding the items from that...